staging/src/k8s.io/apiserver/pkg/storage/value/metrics.go

```diff
@@ -41,6 +41,23 @@ var (
 		},
 		[]string{"transformation_type"},
 	)
+	transformerSuccessTotal = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "transformation_success_total",
+			Help:      "Total number of successful transformations.",
+		},
+		[]string{"transformation_type"},
+	)
+	transformerLastOperationStatus = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "transformation_last_status",
+			Help:      "Status of the last transformation.",
```

Comment:
I don't expect last status to tell you very useful information since a successful call will erase the useful data. It seems much more appropriate to get this information by looking at logs. I think error counts and/or high latencies are a better way to get that information.

Reply:
We are not erasing useful data - it will remain in other metrics - latency and success/failure count.

Would not looking at logs be subject to the same set of constraint inherent when dealing with parallel operations.